packages/ember-metal/lib/index.js

```diff
@@ -275,30 +275,6 @@ Ember.isGlobalPath = isGlobalPath;
 
 Ember.run = run;
 
-/**
-@class Backburner
-@for Ember
-@private
-*/
-Ember.Backburner = function() {
-  deprecate(
-    'Usage of Ember.Backburner is deprecated.',
-    false,
-    {
-      id: 'ember-metal.ember-backburner',
-      until: '2.8.0',
```

Comment:
The deprecation needs to be included in 2.8.0, which is what master is tracking.